src/get-variables.js

```diff
@@ -1,7 +1,7 @@
 import stripComments from 'strip-json-comments';
 
 export default function getVariables(content) {
-  const variableRegex = /\$(.+):\s+(.+);/;
+  const variableRegex = /\$(.+):\s+(.+);?/;
```

Comment:
we could tweak the check so it accounts for a semicolon or a new line so that it's a little stricter